tslint.json

```diff
@@ -11,7 +11,18 @@
         "member-access": [true, "no-public"],
         "no-empty": [true, "allow-empty-catch", "allow-empty-functions"],
         "no-string-literal": false,
-        "interface-over-type-literal": false,
+        "interface-over-type-literal": true,
+        "arrow-return-shorthand": true,
+        "no-angle-bracket-type-assertion": true,
+        "no-redundant-jsdoc": true,
+        "no-unnecessary-initializer": true,
+        "no-unsafe-finally": true,
+        "object-literal-shorthand": true,
+        "prefer-for-of": true,
+        "prefer-readonly": true,
```

Comment:
In practice this won't come up much because we don't explicitly mark things as private, but I still don't know how much I agree with this. Like `prefer-const`, I think there's a difference between "this isn't currently modified by anything" and "this shouldn't be modified".